### PR TITLE
Get rid of webdeployment-common

### PR DESCRIPTION
**Pull request by damccorm**
**Description**: Move webdeployment-common into the only task that uses it

**Documentation changes required:** N

**Added unit tests:** N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/AzureRmWebAppDeploymentV3/make.json [hunk 1]
```diff
@@ -25,13 +17,37 @@
         {
             "source": "postDeploymentScript",
             "options": "-R"
+        },
```
Comment (stephenmichaelf) [inline Tasks/AzureRmWebAppDeploymentV3/make.json:20]: What are these for?

File: Tasks/AzureRmWebAppDeploymentV3/webdeployment-common/deployusingmsdeploy.ts [hunk 2]
```diff
@@ -74,7 +74,7 @@ export async function DeployUsingMSDeploy(webDeployPkg, webAppName, publishingPr
     finally {
         process.env.PATH = pathVar;
         if(setParametersFile != null) {
-            tl.rmRF(setParametersFile, true);
+            tl.rmRF(setParametersFile);
```
Comment (stephenmichaelf) [inline Tasks/AzureRmWebAppDeploymentV3/webdeployment-common/deployusingmsdeploy.ts:77]: Did we mean to do this?

File: Tasks/AzureRmWebAppDeploymentV3/webdeployment-common/ziputility.ts [hunk 3]
```diff
@@ -9,7 +9,7 @@ var archiver = require('archiver');
 export async function unzip(zipLocation, unzipLocation) {
     var defer = Q.defer();
     if(tl.exist(unzipLocation)) {
-      tl.rmRF(unzipLocation, false);
+      tl.rmRF(unzipLocation);
```
Comment (stephenmichaelf) [inline Tasks/AzureRmWebAppDeploymentV3/webdeployment-common/ziputility.ts:12]: Did we mean to do this?

Comment (damccorm) [reply Tasks/AzureRmWebAppDeploymentV3/webdeployment-common/ziputility.ts:12]: Same as previous comment

File: Tasks/AzureRmWebAppDeploymentV3/make.json (hunk 1 above)
Comment (damccorm) [reply Tasks/AzureRmWebAppDeploymentV3/make.json:20]: We need to copy some non-TypeScript files over to the _build directory. Those originally just got baked into the node_module when the common package was compiled

**Review by stephenmichaelf — APPROVED** [review]

**Pull request merged**